tools/prom-metrics-collector/metrics_collector.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	parser "github.com/kubevirt/monitoring/pkg/metrics/parser"
+	"kubevirt.io/ssp-operator/internal/operands/metrics"
+
+	dto "github.com/prometheus/client_model/go"
+)
+
+var excludedMetrics = map[string]struct{}{
+	"kubevirt_ssp_operator_up_total":           {},
+	"kubevirt_ssp_template_validator_up_total": {},
+}
+
```

Comment:
Add comment // ReadMetrics read and parse the metrics to a MetricFamily

Reply:
I actually prefer the code to be self-explained, do you think we should rename the function?

Reply:
Since we try to make it as much similar in all operator I think the name can stay ReadMetrics.
if you strongly against the comment it's OK with me to leave as is, but I think it's nice to have.